CHANGELOG.md

```diff
@@ -487,7 +488,7 @@ convenience to users arriving from outdated links.
   Due to incorrect use of a default URL, singularity action commands
   (run/shell/exec) specifying a container using a library:// URI will always
   attempt to retrieve the container from the default remote endpoint
-  (cloud.sylabs.io) rather than the configured remote endpoint. An attacker may
+  (cloud.sycloud.io) rather than the configured remote endpoint. An attacker may
```

Comment:
Not sure it makes sense to update links in the _changelog_, if this is expected to be rolled back?